typescript/sdk/src/events.ts

```diff
@@ -168,14 +168,14 @@ async function getPaginatedEvents<T extends TypedEvent>(
 ): Promise<Array<T>> {
```

Comment:
Note, I plan to delete this file entirely when we move SmartProvider to the SDK. Though I'm open to just killing it now if people prefer.